extra/3-magic-8-ball.js

```diff
@@ -45,8 +46,34 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+let answers = [
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it.",
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes.",
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again.",
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful.",
+];
+let answer = "";
+
 function shakeBall() {
   //Write your code in here
+  console.log("The ball has shaken!");
+  return answer = answers[Math.floor(Math.random() * answers.length)];
```

Comment:
There's a lot going on in this line, could break it into several, similar to your 'good code' example above?